src/test/cbmc/Assume/main_fail.rs

```diff
@@ -9,8 +9,12 @@ fn __nondet<T>() -> T {
     unimplemented!()
 }
 
+fn __VERIFIER_expect_fail(cond: bool, msg: &str)  {
+    unimplemented!()
+}
+
```

Comment:
This should be included in the new RMC prelude.

Reply:
Willdo